rpcd/playbooks/stage-python-artifacts.yml

```diff
@@ -0,0 +1,184 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Stage the git and python artifacts
+  hosts: "{{ staging_host | default('localhost') }}"
+  user: root
+  vars:
+    upstream_artifact_base_url: "{{ rpco_mirror_base_url | default('https://rpc-repo.rackspace.com') }}"
```

Comment:
The intent with this playbook is to be able to execute it regardless of whether any configuration has been bootstrapped. This allows the staging to be done, for example, onto a laptop or something. That's why I set it to use ``rpco_mirror_base_url`` if it's available, but provide a default. Once the artifacts are staged to a laptop/USB key the expectation is that the deployer would copy the contents over to the target using rsync or whatever.